app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -34,26 +34,56 @@ tm-page(:title='validatorTitle(this.validator)')
       tm-list-item(dt="Commission Maximum" :dd="pretty(validator.commission_max) + ' %'")
       tm-list-item(dt="Commission Change-Rate" :dd="pretty(validator.commission_change_rate) + ' %'")
       tm-list-item(dt="Commission Change Today" :dd="pretty(validator.commission_change_today) + ' %'")
+
+    modal-stake(
+      v-if="showModalStake"
+      v-on:stake="onStake"
+      :showModalStake.sync="showModalStake"
+      :fromOptions="[{ key: `My Wallet - ${this.wallet.address}`, value: 0 }]"
+      :maximum="this.totalAtoms - this.oldBondedAtoms"
+      :to="validator.owner"
+    )
+
+    tm-btn(
+      @click.native="showModalStake = true"
+      color="primary"
+      value="Stake"
+    )
 </template>
 
 <script>
 import { mapGetters } from "vuex"
-import { TmListItem, TmPage, TmPart, TmToolBar } from "@tendermint/ui"
+import { TmBtn, TmListItem, TmPage, TmPart, TmToolBar } from "@tendermint/ui"
 import { TmDataError } from "common/TmDataError"
+import ModalStake from "staking/ModalStake"
 import numeral from "numeral"
 import AnchorCopy from "common/AnchorCopy"
 export default {
   name: "page-validator",
   components: {
     AnchorCopy,
+    "modal-stake": ModalStake,
+    TmBtn,
     TmListItem,
     TmPage,
     TmPart,
     TmToolBar,
     TmDataError
   },
+  data: () => ({
+    showModalStake: false
+  }),
   computed: {
-    ...mapGetters(["delegates", "config", "keybase"]),
+    ...mapGetters([
+      `bondingDenom`,
```

Comment:
Do we need a linter rule for strings without interpolation?

Reply:
What would the rule do?

Reply:
Decide on single quotes on strings without interpolation?

Reply:
As a matter of habit I always use backquotes, otherwise I have to go through this less efficient process:

1.  Writing a string with single quotes.
1.  Realize that I want to add an interpolation.
1.  Add the interpolation.
1.  Change the single quotes to backquotes.

Starting with backquotes means I can skip step #4.  It happens often enough to save me significant time.

Reply:
How about this rule: https://eslint.org/docs/rules/quotes which would auto fix all quotes to backticks where possible?

Reply:
I personally prefer to have double quotes everywhere and use backquotes only when using interpolation